woocommerce/woocommerce-android
# Activate: Login button swap experiment

This PR adds a new login improvement experiment, which swaps the login CTA buttons (`Continue with WordPress.com` and `Enter your store address`).

![image](https://user-images.githubusercontent.com/1522856/184649692-21dc0cab-08ea-4bf6-860a-7e5c64433a01.png)

**To test:**

1. Comment out the contents of the `FirebaseRemoteConfigRepository::fetchRemoteConfig()` and change the `LoginButtonSwapVariant.CONTROL.name` to `LoginButtonSwapVariant.SWAPPED.name` of `defaultValues` in the same class
2. Start the app (skip the prologue carousel if it's shown) and get to the login prologue screen
3. Verify that `Continue with WordPress.com` is the primary button and `Enter your store address` is a secondary button
4. Change the `LoginButtonSwapVariant.SWAPPED.name` to `LoginButtonSwapVariant.CONTROL.name` 
5. Run the app again and verify that `Enter your store address` is the primary button and `Continue with WordPress.com` is a secondary button
